Leroi/Leroi.f90

```diff
@@ -2768,7 +2768,7 @@ SUBROUTINE READ_SYSTEM_AND_SURVEY_DATA
  9 FORMAT(/T12,'Frequency      Transmitter current in amps', &
           /T12,'---------      ---------------------------'/)
  10 FORMAT(//T3,'SURVEY_TYPE =',I2)
- 11 FORMAT(/T3,'NLINES =',I3,';   MRXL =',I4,';   NTX =',I3,';   SOURCE_TYPE =',I2,';   MXVRTX =',I2,';   NTRN =',I3)
+ 11 FORMAT(/T3,'NLINES =',I3,';   MRXL =',I4,';   NTX =',I3,';   SOURCE_TYPE =',I2,';   MXVRTX =',I4,';   NTRN =',I3)

```

Comment:
I4 is allotting an integer with 4 spaces

Reply:
do the integers need to be 0 or space padded (e.g. `0014` of ` 14`)